blog-posts/howto-blog-post.md

```diff
@@ -166,6 +184,16 @@ footer p {
 
 ```
 
+#### Code Block Caption
+
+A code block caption can be added with:
+```html
+<figure>
+  
+  // Code Block
+  <figcaption>Code Block 1: This is an example for a caption.</figcaption>
+</figure>
+```
```

Comment:
I don't see an example for this in the markdown being rendered which would make sense here, wouldn't it ?